README.md

```diff
@@ -18,8 +18,12 @@ Command              | Result
 
 &#9733; in the event browsersync throws a permissions error, `chown` `node_modules/`
 
-### Releases
+### Deprecated as of v1.5.14:
+* `.display--text2`, `.display--text2`
```

Comment:
- `display--text2` listed twice; should this be `text--display2`?
- are the deprecations breaking? It would be good to mention if these deprecated classes will still be supported in this release